src/main/java/org/yorm/util/RowRecordConverter.java

```diff
@@ -1,90 +1,343 @@
 package org.yorm.util;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.yorm.exception.YormException;
+
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodType;
 import java.math.BigDecimal;
-import java.sql.Date;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Time;
-import java.sql.Timestamp;
+import java.sql.*;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
-import org.yorm.exception.YormException;
+import java.util.Map;
 
 public class RowRecordConverter {
+    private static String test =
```

Comment:
I guess this String test is no longer needed, isn't it?

Reply:
Yup! That's removed as part of the second commit that I apparently never pushed, haha!